### DISCUSSION TITLE
How to handle a Field which has an unknown JSON structure?

### DISCUSSION THREAD

**Discussion by esviad** [pydantic/pydantic#5321] [Question] [answered]
I have to model a JSON structure where one of the keys can have a dynamic JSON structure. 
E.g., the structure can look like this: 
{
"request-id": 123,
"status": "completed",
"original-request": {"key-a": 123, "key-b": [1, 2, 3]}
}
or like this:
{
"request-id": 234,
"status": "completed",
"original-request": {"foo": "bar"}
}

The "original-request" field can vary depending on certain conditions. These variations are not in a limited set. How do I model this using Pydantic? 
I don't actually need the original request field or care if validation fails for it, but I need the rest of the data.

**Comment by dmontagu** [comment, marked as answer]
Does this work for you? 
```python
    ...
    original_request: typing.Any = Field(alias='original-request')
```

**Reply by esviad** [reply]
Thank you! This ought to do it.
